hack/verify-govet.sh

```diff
@@ -1,83 +1,19 @@
 #!/bin/bash
 
+set -o errexit
 set -o nounset
 set -o pipefail
 
-GO_VERSION=($(go version))
-
-if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.[4-5]') && -z "${FORCE_VERIFY-}" ]]; then
-  echo "Unknown go version '${GO_VERSION}', skipping go vet."
-  exit 0
-fi
+echo $(go version)
 
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
-source "${OS_ROOT}/hack/common.sh"
-source "${OS_ROOT}/hack/util.sh"
 
 cd "${OS_ROOT}"
-mkdir -p _output/govet
-
-os::build::setup_env
-
-FAILURE=false
-test_dirs=$(find_files | cut -d '/' -f 1-2 | sort -u)
-for test_dir in $test_dirs
-do
-  go tool vet -shadow=false $test_dir
```

Comment:
Those are not techincally false positives... there is an `err` declared in the outer scope and we just declared another one in the inner scope -- yes, the idiomatic error handling sometimes forces us to do that, but that doesn't mean the warning is invalid.

We saw all of those messages on 1.5, they're not new to 1.6. The Travis flakes are that - flakes. I haven't been able to determine why it insists on doing shadow tests on packages in the blacklist, but when it does, it reports the (valid) errors.